libraries/Microsoft.Bot.Builder/IPropertyContainer.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Microsoft.Bot.Builder
+{
+    public interface IPropertyContainer
```

Comment:
I find the name "Property" to be far too abstract to stand on its own. At bare minimum, how about `IStatePropertyContainer`?